### PR TITLE
Add user data password policy rule

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-12086

### CODE REVIEW

File: camunda-commons-ui/lib/widgets/password/cam-widget-password.html [hunk 1]
```diff
@@ -1,4 +1,16 @@
 <div class="password-input right-addon">
+  <script type="text/ng-template" id="popover">
+      <p>{{ tooltip | translate }}</p>
+
+      <ul>
+        <li ng-repeat="rule in rules"
+            translate="{{ rule.placeholder }}"
+            translate-values="{{ rule.parameter }}"
+            ng-if="!!!rule.valid">
```
Comment (marstamm) [inline camunda-commons-ui/lib/widgets/password/cam-widget-password.html:9]:
```suggestion
            ng-if="!rule.valid">
```

File: camunda-commons-ui/lib/widgets/password/cam-widget-password.js [hunk 2]
```diff
@@ -81,49 +69,68 @@ module.exports = [
...
 
         $scope.$watch('[password]', handlePasswordUpdate, true);
+
+        function sanitize(value) {
+          if (value === undefined) {
+            return '';
+          }
+
+          return value === null ? '' : value;
+        }
```
Comment (marstamm) [inline camunda-commons-ui/lib/widgets/password/cam-widget-password.js:133]: Let's move this function somewhere before `passwordPolicyProvider.validate`. When reading the code, we then see it defined before we use it.

undefined and null are both false-y values, so we can shorten this function to 
```suggestion
        function sanitize(value) {
          return value || '';
        }
```

or, for added style points:
```suggestion
        var sanitize = value => value || '';
```

File: camunda-commons-ui/lib/widgets/password/cam-widget-password.html (hunk 1 above)
Comment (tasso94) [reply camunda-commons-ui/lib/widgets/password/cam-widget-password.html]: 👍

File: camunda-commons-ui/lib/widgets/password/cam-widget-password.js (hunk 2 above)
Comment (tasso94) [reply camunda-commons-ui/lib/widgets/password/cam-widget-password.js]: 👍

**Pull request merged**